drones/connection/__init__.py

```diff
@@ -0,0 +1,9 @@
+"""Library to manage communication with Tello drone.
+
+Additional info.
+"""
+
+# Import prototype and share it as global object
+from drones.connection.prototype import ConnectorPrototype
+
+c = ConnectorPrototype()

```

Comment:
In general, you should not declare variables in `__init__.py` files since they are purely for defining module imports layout. I suggest deleting this line. Please refer to further review comments on how to resolve this.